packages/sanity/src/structure/panes/document/DocumentPaneContext.ts

```diff
@@ -67,7 +67,6 @@ export interface DocumentPaneContextValue {
   setTimelineMode: (mode: TimelineMode) => void
   setTimelineRange(since: string | null, rev: string | null): void
   setIsDeleting: (state: boolean) => void
-  source?: string
```

Comment:
Removing it as it's never defined in the context value given to `DocumentPaneContext` and seems to have been this way for at least the last 2 years